### ISSUE TITLE
6.0-beta1 changes naming rules for nested childs pages 

### ISSUE THREAD

**Issue by qwazer** [bsorrentino/maven-confluence-plugin#156]
HI
 I update from 5.1.1 to 6.0-beta1

I have such site description
```
    <home name="E2E SDK" uri="../markdown/parent.md" ignore-variables="true">

        <child name="01. Introduction" uri="../markdown/dummy.md">
            <child name="Brief Overview" uri="../markdown/02-brief-overview.md"/>
            <child name="History" uri="../markdown/05-history.md"/>
        </child>
```

previously it generates page with name
E2E SDK - Brief Overview

now it generate page with name

E2E SDK - 01. Introduction - Brief Overview

please keep backward-compatibility or add option to have the same naming generation rule for nested child pages

**Comment by bsorrentino** [comment]
Hi thanks for feedback 

I'll fix it as soon as possible

**Comment by bsorrentino** [comment]
I've released the `6.0-SNAPSHOT` ... can you check it and let me know if issue is fixed as requested ?

**Comment by qwazer** [comment]
@bsorrentino Looks like snapshot version 6.0-SNAPSHOT is not available. I tried
```
        <pluginRepository>
          <id>sonatype</id>
          <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
          <releases><enabled>false</enabled></releases>
          <snapshots><enabled>true</enabled><updatePolicy>always</updatePolicy></snapshots>
        </pluginRepository>
      </pluginRepositories>
```

**Comment by bsorrentino** [comment]
@qwazer  your configuration looks good .... it is very strange because I’ve checked and the '6.0-SNAPSHOT' is there

**Comment by bsorrentino** [comment]
fix released in version `6.0-beta2`

**Comment by qwazer** [comment]
@bsorrentino thanks!!!
